opensrp-chw/src/miecd/java/org/smartregister/chw/util/UtilsFlv.java

```diff
@@ -0,0 +1,60 @@
+package org.smartregister.chw.util;
+
+import android.os.AsyncTask;
+import android.view.Menu;
+
+import org.apache.commons.lang3.StringUtils;
+import org.smartregister.chw.R;
+import org.smartregister.chw.core.rule.MalariaFollowUpRule;
+import org.smartregister.chw.core.utils.CoreConstants;
+import org.smartregister.chw.core.utils.MalariaVisitUtil;
+import org.smartregister.chw.dao.MalariaDao;
+import org.smartregister.chw.fp.dao.FpDao;
+import org.smartregister.util.Utils;
+
+import java.util.Date;
+
+public class UtilsFlv {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'UtilsFlv' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426631)